actor/src/main/scala/org/apache/pekko/actor/FSM.scala

```diff
@@ -743,29 +743,29 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
     case x    => x.stateData
   }
 
-  /*
+  /**
    * ****************************************************************
    *                PRIVATE IMPLEMENTATION DETAILS
    * ****************************************************************
    */
 
   private[pekko] def debugEvent: Boolean = false
 
-  /*
+  /**
```

Comment:
actually, I don't see the point in changing the field comments

Can you just change function comments and only function comments?

Reply:
ok, I will check it again